src/test/java/com/example/hubtjobsreactive/security/WithMockJwt.java

```diff
@@ -0,0 +1,20 @@
+package com.example.hubtjobsreactive.security;
+
+import org.springframework.security.test.context.support.WithSecurityContext;
+
+import java.lang.annotation.*;
+
+@Target({ElementType.METHOD, ElementType.TYPE})
+@Retention(RetentionPolicy.RUNTIME)
+@Inherited
+@Documented
+@WithSecurityContext(factory = WithMockJwtSecurityContextFactory.class)
+public @interface WithMockJwt {
```

Comment:
WithMockUser still have String[] roles() default { "USER" }
we don't need to create this custom annotation

Reply:
The @WithMockUser provides a org.springframework.security.core.userdetails.User object when the code requires the Jwt type, that's why I have to create a customer annotation for that need